src/unity/python/turicreate/toolkits/image_classifier/image_classifier.py

```diff
@@ -53,6 +69,78 @@ def create(dataset, target, feature = None, model = 'resnet-50',
         Name of the column containing the input images. 'None' (the default)
         indicates the only image column in `dataset` should be used as the
         feature.
+        
+l2_penalty : float, optional
+        Weight on l2 regularization of the model. The larger this weight, the
+        more the model coefficients shrink toward 0. This introduces bias into
+        the model but decreases variance, potentially leading to better
+        predictions. The default value is 0.01; setting this parameter to 0
+        corresponds to unregularized logistic regression. See the ridge
+        regression reference for more detail.
+
+    l1_penalty : float, optional
+        Weight on l1 regularization of the model. Like the l2 penalty, the
+        higher the l1 penalty, the more the estimated coefficients shrink toward
+        0. The l1 penalty, however, completely zeros out sufficiently small
+        coefficients, automatically indicating features that are not useful
+        for the model. The default weight of 0 prevents any features from
+        being discarded. See the LASSO regression reference for more detail.
+
+    solver : string, optional
+        Name of the solver to be used to solve the regression. See the
+        references for more detail on each solver. Available solvers are:
+
+        - *auto (default)*: automatically chooses the best solver for the data
+          and model parameters.
+        - *newton*: Newton-Raphson
+        - *lbfgs*: limited memory BFGS
+        - *fista*: accelerated gradient descent
+
+        For this model, the Newton-Raphson method is equivalent to the
+        iteratively re-weighted least squares algorithm. If the l1_penalty is
+        greater than 0, use the 'fista' solver.
+
+        The model is trained using a carefully engineered collection of methods
+        that are automatically picked based on the input data. The ``newton``
+        method  works best for datasets with plenty of examples and few features
+        (long datasets). Limited memory BFGS (``lbfgs``) is a robust solver for
+        wide datasets (i.e datasets with many coefficients).  ``fista`` is the
+        default solver for l1-regularized linear regression. The solvers are all
+        automatically tuned and the default options should function well. See
+        the solver options guide for setting additional parameters for each of
+        the solvers.
+
+        See the user guide for additional details on how the solver is chosen.
+        (see `here
+        <https://apple.github.io/turicreate/docs/userguide/supervised-learning/linear-regression.html>`_)
+
+    feature_rescaling : boolean, optional
+
```

Comment:
For consistency sake, let's not have a blank line between the parameter name/type and description.

Reply:
@TobyRoseman Done.